packages/relay/tests/lib/eth/eth_getBlockByNumber.spec.ts

```diff
@@ -566,4 +566,66 @@ describe('@ethGetBlockByNumber using MirrorNode', async function () {
       args,
     );
   });
+
+  it('eth_getBlockByNumber should skip transactions with wrong nonces when showDetails = false', async function () {
```

Comment:
Aren't these two test cases basically the same module `showDetails=true|false`? If that's the case, maybe it's worth to merge them into a single test case, something like

```
[false, true].forEach(showDetails => {
  it(`eth_getBlockByNumber should skip transactions with wrong nonces when showDetails = ${showDetails}`, async function () {
  });
});
```

Reply:
Fair point, that would simplify things a bit